.github/workflows/notify-teams-channel.yml

```diff
@@ -7,12 +7,13 @@ on:
     types: [opened, reopened]
 
 jobs:
-  notify:
+  notify-teams-channel:
+    name: post
     runs-on: ubuntu-latest
     
     steps:
       - name: Notify
-        uses: davidwengier/PostAdaptiveCard@v1.0.4
+        uses: davidwengier/postadaptivecard@ed077f2bd4b80fdaaef21dae2e13a34e556d466d
```

Comment:
Where does this SHA come from?

Reply:
It is the commit SHA that relates to that version of the action. I mentioned the lack of user-friendliness of accessing this in my email thread with the .NET OSS Admins.